langchain4j-core/src/main/java/dev/langchain4j/data/message/AudioContent.java

```diff
@@ -0,0 +1,130 @@
+package dev.langchain4j.data.message;
+
+import dev.langchain4j.data.audio.Audio;
+
+import java.net.URI;
+import java.util.Objects;
+
+import static dev.langchain4j.data.message.ContentType.AUDIO;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+
+public class AudioContent implements Content {
```

Comment:
Please update these comments in `UserMessage`:
`Content can be either TextContent or ImageContent.`